app/client/sagas/userSagas.js

```diff
@@ -154,17 +158,20 @@ function* resetPin({ payload }) {
 
     yield put({ type: RESET_PIN_SUCCESS, reset_user: reset_response });
 
-    yield put({
-      type: ADD_FLASH_MESSAGE,
-      error: false,
-      message: reset_response.message
-    });
+    yield put(
+      MessageAction.addMessage({
+        error: false,
+        message: reset_response.message
+      })
+    );
   } catch (fetch_error) {
     const error = yield call(handleError, fetch_error);
```

Comment:
we should consider adding the following code to the `handleError` function in utils.js: `store.dispatch(MessageAction.addMessage({ error: true, message: error.message }))`

Reply:
though that does presume we necessarily want to flash the error (sometimes it needs interpretation i think?). Guess we could also add that as an input arg...